readme.rst

```diff
@@ -163,44 +163,53 @@ Documentation in a code repository
-#. in file ``docs/.readthedocs.yaml`` set the following:
+#. Create a directory called :file:`docs` at the root of the code repository.
+#. Populate the above directory with the contents of the starter pack
+   repository (with the exception of the :file:`.git` directory).
+#. Copy the file(s) located in the :file:`docs/.github/workflows` directory into
+   the :file:`.github/workflows` directory in the root of the code repository.
+#. In the above workflow file(s), change the value of the
+   :file:`working-directory` field from ``.`` to ``docs``.
+#. Create a symbolic link to the :file:`docs/.wokeignore` file from the root
+   directory of the code repository.
+#. In the :file:`docs/.readthedocs.yaml` file, set the following:
 
    * ``post_checkout: cd docs && python3 .sphinx/build_requirements.py``
    * ``configuration: docs/conf.py``
    * ``requirements: docs/.sphinx/requirements.txt``
 
-**Note:** When configuring RTD itself for your project, the setting "Path for
-``.readthedocs.yaml``" (under **Advanced Settings**) will need to be given the
+**Note:** When configuring RTD itself for your project, the setting \"Path for
+``.readthedocs.yaml``\" (under **Advanced Settings**) will need to be given the
 value of ``docs/.readthedocs.yaml``.
 
 Automation
 ^^^^^^^^^^
 
 To automate the initialisation for either scenario ensure you have the following:
 
-- A GitHub repository where you want to host your documentation, cloned to your local machine. The recommended approach is to host the documentation alongside your code in a ``docs`` folder. But a standalone documentation repository is also an option; in this case, start with an empty repository.
+- A GitHub repository where you want to host your documentation, cloned to your
+  local machine. The recommended approach is to host the documentation alongside
+  your code in a :file:`docs` folder. But a standalone documentation repository
+  is also an option; in this case, start with an empty repository.
 - Git and Bash installed on your system.
 
-There is a provided ``init.sh`` Bash script that does the following:
+There is a provided :file:`init.sh` Bash script that does the following:
 
-- clones the starter pack GitHub repository
-- creates the specified installation directory if necessary
-- updates working directory paths in workflow files, and updates configuration paths in the ``.readthedocs.yaml`` file
-- copies and moves contents and ``.github`` files from the starter pack to the installation directory
-- deletes the cloned repository when it's done
+- Clones the starter pack GitHub repository,
+- Creates the specified installation directory (if necessary),
+- Updates working directory paths in workflow files, and updates configuration
+  paths in the :file:`.readthedocs.yaml` file,
+- Copies and moves contents and :file:`.github` files from the starter pack to
+  the installation directory, and
+- Deletes the cloned repository when it\'s done.
```

Comment:
```suggestion
- Clones the starter pack GitHub repository.
- Creates the specified installation directory (if necessary).
- Updates working directory paths in workflow files, and updates configuration
  paths in the :file:`.readthedocs.yaml` file.
- Copies and moves contents and :file:`.github` files from the starter pack to
  the installation directory.
- Deletes the cloned repository when it\'s done.
```

I'm very opposed to adding commas and "and"s between list items. Imo, list items should all be formed consistently. Also, this can cause issues for translation (which isn't relevant here, but I still generally adhere to that - readers might use automatic translation).